GNUmakefile

```diff
@@ -1,12 +1,9 @@
-GOTOOLS = github.com/mitchellh/gox golang.org/x/tools/cmd/stringer \
```

Comment:
I think this set of things / associated target is useful for people building for the first time. These aren't built into the binary so we don't want to vendor them, but they are Go-based tools used in the build process.